arch/x64/arch-elf.hh

```diff
@@ -42,4 +42,37 @@ enum {
 
 #define ELF_KERNEL_MACHINE_TYPE 62
 
+#define SAFETY_BUFFER 256
```

Comment:
If this macro isn't for **users** of elf.hh but just for the internal implementation below, please make it a constexpr inside the function instead of a macro, so it doesn't "leak" outside this implementation.

Reply:
Addressed in the updated PR.